### PR TITLE
Included config file prefixes in render_env

### PR DESCRIPTION
**Pull request by MahatmaCane**

### CODE REVIEW

File: conda_rpms/build_rpm_structure.py [hunk 1]
```diff
@@ -137,7 +143,8 @@ def create_rpmbuild_for_tag(repo, tag_name, target, config):
 
     pkgs = [pkg for _, pkg in manifest]
     env_name, tag = tag_name.split('-')[1:]
-    with open(os.path.join(target, 'SPECS', '{}-env-{}-tag-{}.spec'.format(rpm_prefix, env_name, tag)), 'w') as fh:
+    s = '{}-env-{}-tag-{}.spec'.format(rpm_prefix, env_name, tag)
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py]: Perhaps name this `fname`?

File: conda_rpms/build_rpm_structure.py [hunk 2]
```diff
@@ -163,21 +171,12 @@ def create_rpmbuild_content(repo, target, config):
...
-
-                env = '{}-{}'.format(branch.name, label)
-                tags.append(tag)
-                envs.append(env)
-                # If we wanted RPMS for each label, enable this.
-#                create_rpmbuild_for_label(env, tag, target)
-            for tag in tags:
                 create_rpmbuild_for_tag(repo, tag, target, config)
-
+                s = 'SciTools-env-{}-label-{}.spec'.format(branch.name, label)
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py]: Ditto. `fname` works.

File: conda_rpms/build_rpm_structure.py [hunk 3]
```diff
@@ -163,21 +171,12 @@ def create_rpmbuild_content(repo, target, config):
             #--------------- New for this ---------
 
             # Keep track of the labels which have tags - its those we want.
-            tags = []
-            envs = []
             for label, tag in labelled_tags.items():
-                deployed_name = tag.split('-', 2)[2]
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py:169]: Would you just mind talking me through the logic of removing these lines?

File: conda_rpms/generate.py [hunk 4]
```diff
@@ -48,6 +49,18 @@ def render_dist_spec(dist, config):
                                 rpm_prefix=rpm_prefix,
                                 install_prefix=install_prefix)
 
+def render_env(branch_name, label, repo, config, tag):
+    env_info = {'url': 'http://link/to/gh',
+                'name': branch_name,
+		        'label' : label,
```
Comment (pelson) [inline conda_rpms/generate.py]: Is there a tab character in here?

File: conda_rpms/build_rpm_structure.py (hunk 3 above)
Comment (MahatmaCane) [inline conda_rpms/build_rpm_structure.py:169]: Looks like lines 169-171 are unused. Then looks like you're adding to the tags and envs list with each iteration only to loop through those lists again to call create_rpmbuild_for_tag. I think I've just tidied it up - have I deleted something important?

Comment (pelson) [inline conda_rpms/build_rpm_structure.py:169]: Only the optimisation that prevents tags from being rendered multiple times.

**Comment by pelson** [comment]
Typically we try to create separate PRs for stylistic changes and features as it enables to iterate more rapidly, but in this case it didn't have too much of a detrimental impact, so I'm definitely :+1: 

Thanks @MahatmaCane.

**Pull request merged**